tests/e2e/actions.go

```diff
@@ -2065,7 +2073,7 @@ func (tr TestConfig) invokeDoublesignSlash(
 		if err != nil {
 			log.Fatal(err, "\n", string(bz))
 		}
-		tr.waitBlocks("provi", 10, 2*time.Minute)
+		tr.waitBlocks("provi", 20, 4*time.Minute)
```

Comment:
Just a note for posterity: Increased this due to some flakiness in the test involving double signing.